lang/en.js

```diff
@@ -515,6 +517,7 @@ export default {
   "results": "Results",
   "resultsLimitWarning": "Additional results are not shown as only the first 1000 most relevant results are shown. If you haven't found what you're looking for, please consider refining your search.",
   "search": "Search",
+  "searchResults": "Search results for ",
```

Comment:
use interpolation here like so:
```
"Search results for '{0}'",
```
And pass the search term into the string.

Reply:
ok